File: data/src/main/java/com/github/gmetal/data/entity/GuestSessionEntity.java
package com.github.gmetal.data.entity;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject
public class GuestSessionEntity extends BaseMoviesDbEntity {

    @JsonField
    public boolean success;

    @JsonField(name = "guest_session_id")
    public String guestSessionId;

    @JsonField(name = "expires_at")
    public String expiresAt;
}

File: data/src/main/java/com/github/gmetal/data/entity/GuestSessionEntity.kt
package com.github.gmetal.data.entity

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class GuestSessionEntity : BaseMoviesDbEntity() {

    var success: Boolean = false

    @SerialName("guest_session_id")
    var guestSessionId: String? = null

    @SerialName("expires_at")
    var expiresAt: String? = null
}

